### ISSUE TITLE
[Tickets] Exception in command 'tickets addmodal' (Embed size exceeds maximum size of 6000)

### ISSUE THREAD

**Issue by Skyridr** [vertyco/vrt-cogs#152]
So was trying to add a modal to a ticket both short and long, and I keep receiving this:

```py
Exception in command 'tickets addmodal'
Traceback (most recent call last):
  File "/home/koala/ticketmaster/lib/python3.11/site-packages/discord/ext/commands/core.py", line 235, in wrapped
    ret = await coro(*args, **kwargs)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/koala/ticketmaster/cogs/CogManager/cogs/tickets/commands/admin.py", line 389, in addmodal
    await self.create_or_edit_modal(ctx, panel_name, field_name)
  File "/home/koala/ticketmaster/cogs/CogManager/cogs/tickets/commands/admin.py", line 613, in create_or_edit_modal
    await self.initialize(ctx.guild)
  File "/home/koala/ticketmaster/cogs/CogManager/cogs/tickets/tickets.py", line 135, in initialize
    return await self._init_guild(target_guild, data)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/koala/ticketmaster/cogs/CogManager/cogs/tickets/tickets.py", line 165, in _init_guild
    new_id = await update_active_overview(guild, data)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/koala/ticketmaster/cogs/CogManager/cogs/tickets/utils.py", line 387, in update_active_overview
    await message.edit(embeds=embeds)
  File "/home/koala/ticketmaster/lib/python3.11/site-packages/discord/message.py", line 2275, in edit
    data = await self._state.http.edit_message(self.channel.id, self.id, params=params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/koala/ticketmaster/lib/python3.11/site-packages/discord/http.py", line 745, in request
    raise HTTPException(response, data)
discord.errors.HTTPException: 400 Bad Request (error code: 50035): Invalid Form Body
In embeds: Embed size exceeds maximum size of 6000

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/koala/ticketmaster/lib/python3.11/site-packages/discord/ext/commands/bot.py", line 1350, in invoke
    await ctx.command.invoke(ctx)
```

![image](https://github.com/vertyco/vrt-cogs/assets/6794064/129477b9-9c1f-42e9-840f-1d03881b9032)

    
Even when I set a minimal and max way below its limit in the modal setup, I keep getting it. Only additional things I can mention is that the interactions of both ticket buttons now fail even though I only attempted to add the modal to 1 ticket, and that we currently have 94 tickets open with an active-thread overview currently split into 2 embeds. close button doesn't work anymore either.

This also pops up when I reloaded the bot/cog:

```py
[2023-10-20 10:09:45] [ERROR] red.vrt.tickets: Failed to initialize tickets for Age Of Dino
Traceback (most recent call last):
  File "/home/koala/ticketmaster/cogs/CogManager/cogs/tickets/tickets.py", line 146, in initialize
    await self._init_guild(guild, data)
  File "/home/koala/ticketmaster/cogs/CogManager/cogs/tickets/tickets.py", line 165, in _init_guild
    new_id = await update_active_overview(guild, data)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/koala/ticketmaster/cogs/CogManager/cogs/tickets/utils.py", line 387, in update_active_overview
    await message.edit(embeds=embeds)
  File "/home/koala/ticketmaster/lib/python3.11/site-packages/discord/message.py", line 2275, in edit
    data = await self._state.http.edit_message(self.channel.id, self.id, params=params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/koala/ticketmaster/lib/python3.11/site-packages/discord/http.py", line 745, in request
    raise HTTPException(response, data)
discord.errors.HTTPException: 400 Bad Request (error code: 50035): Invalid Form Body
In embeds: Embed size exceeds maximum size of 6000
[2023-10-20 10:09:45] [INFO] red.vrt.tickets: Tickets initialized in 796.7ms
```
Which has to do I assume with the [p]tickets overview command line.

So after I disabled tickets overview, deleted the embed. I was able to create modals again. Guess I found the culprit.

**Comment by vertyco** [comment]
Looks like the combined text of all your modals is more than 6k, i probably won't have time to fix within the next few days

**Comment by vertyco** [comment]
oh nvm it has nothing to do with the addmodal command itself

**Comment by vertyco** [comment]
its because you have too many tickets open and it cant fit them all into one embed

**Comment by vertyco** [comment]
This should be fixed now with v2.4.2
